monitoring/rid_qualifier/host/run_locally.sh

```diff
@@ -11,7 +11,7 @@ fi
 cd "${BASEDIR}/../../.." || exit 1
 
 # This sample assumes that a local DSS instance is available similar to the one
-# produced with /build/dev/run_locally.sh
+# produced with monitoring/rid_qualifier/mock/run_locally.sh
```

Comment:
Running monitoring/rid_qualifier/mock/run_locally.sh produces a mock RID system for use with rid_qualifier (rather than a DSS instance, as the previous line currently says)